repo.go

```diff
@@ -734,13 +770,17 @@ func (r *Repo) AddTargetsWithExpires(paths []string, custom json.RawMessage, exp
 	}); err != nil {
 		return err
 	}
+	return r.WriteTargetWithExpires(t, expires)
+}
+
+func (r *Repo) WriteTargetWithExpires(t *data.Targets, expires time.Time) error {
```

Comment:
Could we make this private, since it lacks the user-friendly validation of the higher-level methods?